sfm/ui/templates/ui/collection_detail.html

```diff
@@ -172,17 +174,26 @@ <h1 class="panel-title">{{ collection.name }}</h1>
                 <thead>
                 <tr>
                     <th>Type</th>
-                    <th>Date requested</th>
+                    <th>Requested</th>
+                    <th>Updated/Completed</th>
                     <th>Status</th>
+                    <th>Stats</th>
                     <th>Messages</th>
                 </tr>
                 </thead>
                 {% for harvest in harvests %}
                 <tr>
                     <td><a href="{% url "harvest_detail" harvest.pk %}">{{ harvest.get_harvest_type_display }}</a></td>
                     <td>{{ harvest.date_requested }}</td>
+                    <td>{% if harvest.status == "completed success" or harvest.status == "completed failure" %}
+                        {{ harvest.date_ended}}
+                        {% else %}
+                            {{ harvest.date_updated|naturaltime }}
+                        {% endif %}
+                    </td>
                     <td>{{ harvest.get_status_display }}</td>
-                    <td>{{ harvest.message_count }} message{{harvest.message_count|pluralize}}</td>
+                    <td>{{ harvest.stats|join_stats }}</td>
+                    <td>{% if harvest.message_count %}{{ harvest.message_count }} message{{harvest.message_count|pluralize}}{% endif %}</td>
```

Comment:
This is blank if there are no items. Can it say 0 items (e.g. 0 tweets) instead? Otherwise it's unclear why there are no stats-- looks like it's still getting the stats, or something went wrong.